frontend/src/api/tests/StreamApi.test.js

```diff
@@ -2,56 +2,163 @@ import axios from "axios";
 import Enzyme from "enzyme";
 import Adapter from "enzyme-adapter-react-16";
 import { afterEach, describe, expect, jest, it } from "@jest/globals";
-// import StreamApi itself
 import * as StreamApi from "../StreamApi";
+import StreamInfo from "../../model/StreamInfo";
+import { convertToDataObject } from "../../model/ConvertDataFormat";
 
 Enzyme.configure({ adapter: new Adapter() });
 
 jest.mock("axios");
 
-const mockSingleStream = {
-  streamId: 1,
-  streamName: "Test 1"
+const axiosAllStreamsResponse = [1, 2];
+const axiosGetFirstStreamResponse = {
+  id: 1,
+  inputChannel: {
+    id: 1,
+    channel: {
+      id: 1,
+      name: "Sample Receiver Channel 1",
+      port: 20002
+    },
+    decoder: {
+      serialNumber: "z7VBn9aK",
+      device: {
+        serialNumber: "z7VBn9aK",
+        publicIpAddress: "0:0:0:0:0:0:0:1",
+        privateIpAddress: "127.0.0.1",
+        displayName: "sample_receiver",
+        status: "online"
+      },
+      input: null
+    }
+  },
+  outputChannel: {
+    id: 2,
+    channel: {
+      id: 4,
+      name: "Sample Sender Channel 2",
+      port: 20001
+    },
+    encoder: {
+      serialNumber: "d7TxFn7o",
+      device: {
+        serialNumber: "d7TxFn7o",
+        publicIpAddress: "0:0:0:0:0:0:0:1",
+        privateIpAddress: "127.0.0.1",
+        displayName: "sample_sender",
+        status: "online"
+      },
+      output: null
+    }
+  },
+  isRendezvous: false
```

Comment:
Maybe we should  start writing fixtures for the frontend tests. The Sample data could be repurposed for that.

Reply:
We already have a DeviceFixture.js, I'll create a StreamFixture.js to house this.